SwiftUI_Homework/SwiftUI_Homework/CheckboxStyle.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CheckboxStyle.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Кастомный Toggle
+struct CheckboxStyle: ToggleStyle {
```

Comment:
```suggestion
struct CheckboxStyle: ToggleStyle {
```
Кажется нужно добавить в название View или ToggleView по названию вообще не понятно что это